packer/scripts/firstuser.sh

```diff
@@ -86,6 +95,8 @@ checkFirstUser() {
    echo "${status}"
 }
 
+stopServices
+cleanServicesPIDs
```

Comment:
thanks, next time, it's preferred (at least to me ) to add the notes *embedded* in code as comment,  instead of PR comments (so that everyone visits this code can understand the reason.)